src/config/consts.js

```diff
@@ -155,7 +155,8 @@ export const consts = {
   LIST_TYPES: {
     CARD_LIST: 'cardList',
     IMAGE_TEXT_LIST: 'imageTextList',
-    TEXT_LIST: 'textList'
+    TEXT_LIST: 'textList',
+    TEXT_WITH_BIG_CARD_LIST: 'textWithBigCardList'
```

Comment:
following the scheme of `XYZ_LIST` here i suggest `CARD_TEXT_LIST`. we do not need a "big" because our card is our card, there is no big or small

Reply:
Yeah, that makes more sense.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/920/commits/ce9321414c47f430d32648b51629f9dd8d6c74ab